apps/engineeringblog/content/site/settings.mdx

```diff
@@ -35,4 +35,6 @@ connect:
       url: https://www.instagram.com/code4africa__/
     - platform: Github
       url: https://github.com/CodeForAfrica
+analytics:
+  analyticsId: "G-TN1WGZVHLE"
```

Comment:
Where can I get more information about this. Does the tsx layout have access to this post compilation as it's an mdx file? A little lost on how the pieces fit together

Reply:
Where is the real `MumaTheProgrammer` and what have you done to him @DavidTheProgrammer ?

  1. The mdx file is loaded [here](https://github.com/CodeForAfrica/ui/blob/442957676567702cd4da7dcba78e00fe809278fd/apps/engineeringblog/lib/data/index.ts#L123-L126), and
  2. It's added to the layout [here](https://github.com/CodeForAfrica/ui/blob/442957676567702cd4da7dcba78e00fe809278fd/apps/engineeringblog/app/layout.tsx#L23-L24).

Reply:
Lol i'll blame it on the flu. But looks good 🚀